frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -1,20 +1,43 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
+  isChecked?: boolean;
   onChange: () => void;
+  isDefaultStyleDisabled?: boolean;
 };
 
-const Checkbox: React.FC<Properties> = ({ label, onChange }) => {
+const Checkbox: React.FC<Properties> = ({
+  label,
+  isChecked,
+  onChange,
+  isDefaultStyleDisabled,
```

Comment:
bad decision, why can't we use default checkbox styles? we should pass other properties like color, isRounded etc. and in case it's something with different layout than use style property as for buttons
```suggestion
```

Reply:
hi, @Liza-Veis Please check these changes, I will try to fix the other comments as soon as I can.